soc/hps_proto2_platform.py

```diff
@@ -79,6 +79,7 @@ def __init__(self, platform, sys_clk_freq):
             o_CLKO=ClockSignal("sys"),
             i_CE=self.sys_clk_enable,
         )
```

Comment:
For my understanding, is the clock of `cd_sys` driven by the clock of `cd_osc` via this instantiation?

Reply:
Indeed, DCC guarantees no glitches and equality in phase of its input clock signal and output signal. According to [Migen documentation](https://m-labs.hk/migen/manual/reference.html#migen.fhdl.structure.ClockSignal), each clock signal retrieved by `ClockSignal()` refers to the same signal so `sys` domain should be identical to `osc` domain but it also has an enable feature.